tensorflow/core/kernels/histogram_op_gpu.cu.cc

```diff
@@ -30,10 +30,9 @@ limitations under the License.
 #include "tensorflow/core/kernels/histogram_op.h"
 #include "tensorflow/core/platform/logging.h"
 #include "tensorflow/core/platform/types.h"
+#include "tensorflow/core/util/gpu_kernel_helper.h"
 #if GOOGLE_CUDA
 #include "tensorflow/core/util/cuda_kernel_helper.h"
```

Comment:
This is incorrect . In GOOGLE_CUDA path there is no cuda_kernel_helper.h

Reply:
Please note that the include for cuda_kernel_helper.h existed before my changes, I simply put it behind the GOOGLE_CUDA macro. So I am not sure why it would be incorrect?

Reply:
cuda_kernel_helper.h is not in this repository anymore

Reply:
I see, did you move it to gpu_kernel_helper.h when porting? Anyways, will replace the header inclusion here and elsewhere then with gpu_kernel_helper.